syz-fuzzer/fuzzer.go

```diff
@@ -202,7 +202,7 @@ func main() {
 	log.Logf(0, "dialing manager at %v", *flagManager)
 	manager, err := rpctype.NewRPCClient(*flagManager, timeouts.Scale)
 	if err != nil {
-		log.Fatalf("failed to connect to manager: %v ", err)
+		log.Fatalf("failed to NewRPCClient() manager: %v ", err)
```

Comment:
Maybe `failed to create an RPC client: %v`?

Reply:
Just done.